Examples/Example2/Example2/ViewController.swift

```diff
@@ -21,7 +21,7 @@ class ViewController: UIViewController {
         
         let carouselFrame = CGRect(x: view.center.x - 200.0, y: view.center.y - 100.0, width: 400.0, height: 200.0)
         carouselView = SwiftCarousel(frame: carouselFrame)
-        carouselView.itemsFactory(itemsCount: 5) { choice in
+        try! carouselView.itemsFactory(itemsCount: 5) { choice in
```

Comment:
Instead of `try!` I was thinking about `do` and `catch` and add a message. 
Something like:
```swift
do {
    try..
} catch {
    fatalError("Oh boy!") // Specific message why the crash happened.
}
```

It would give more specific info and a context for error, so the user would know how to fix it.